src/openrct2/localisation/string_ids.h

```diff
@@ -3647,6 +3647,16 @@ enum {
 	STR_BOOSTER = 5995,
 	STR_RIDE_CONSTRUCTION_BOOSTER_SPEED = 5996,
 
```

Comment:
There are empty lines in the rest of this list, except for count (I don't think it's a bad thing, but it's inconsistent).